.travis.yml

```diff
@@ -22,12 +22,26 @@ matrix:
     env: SKIP_RELEASE=true
   - jdk: oraclejdk9
     env: SKIP_RELEASE=true MOCK_MAKER=mock-maker-inline
+  - install: . ./install-jdk.sh -F 10 -L GPL
+    env: SKIP_RELEASE=true JDK='Oracle JDK 10'
+  - install: . ./install-jdk.sh -F 10 -L GPL
+    env: SKIP_RELEASE=true MOCK_MAKER=mock-maker-inline JDK='Oracle JDK 10'
+  - install: . ./install-jdk.sh -F 11 -L BCL
+    env: SKIP_RELEASE=true JDK='Oracle JDK 11'
+  - install: . ./install-jdk.sh -F 11 -L BCL
+    env: SKIP_RELEASE=true MOCK_MAKER=mock-maker-inline JDK='Oracle JDK 11'
+  allow_failures:
+    - env: SKIP_RELEASE=true JDK='Oracle JDK 11'
+    - env: SKIP_RELEASE=true MOCK_MAKER=mock-maker-inline JDK='Oracle JDK 11'
 
 branches:
   #Don't build tags
   except:
   - /^v\d/
 
+before_install:
+  - wget https://raw.githubusercontent.com/sormuras/bach/master/install-jdk.sh
```

Comment:
I personally don't like to just `wget` files from GitHub in my (release) builds. The file could be downloaded (we lose automatic updates, though). However, I might be exaggerating (and maybe we already "wget" something in the build ;) ).

Reply:
I don't like it either.

I managed a [list of copies](https://sormuras.github.io/blog/2017-12-08-install-jdk-on-travis.html#used-by) this file in various OS projects, but whenever a new feature version of the JDK drops or Oracle changes the path to the binaries, I had to spam pull requests. Some of them are not merged, yet and the projects will be hit by an outdated version of `install-jdk.sh` soon: 
- https://github.com/google/google-java-format/pull/261 
- https://github.com/square/javapoet/pull/629

Reply:
I think, the preferred way to refer to raw files on GitHub is using following URI:

`https://github.com/sormuras/bach/raw/master/install-jdk.sh`

Reply:
@sormuras I took the url stated in https://github.com/junit-team/junit5/blob/13de8c51701b69769328b55b1857631e75bc5cdc/.travis.yml#L13 which is the same as https://github.com/raphw/byte-buddy/blob/a576a7af2e4c9d3621526f40f7faa15fda16eafb/.travis.yml#L8 so I think this is okay?

Reply:
That's the current target URI where you are redirected to when browsing `https://github.com/sormuras/bach/raw/master/install-jdk.sh` -- so, if GitHub changes this to some other address someday, it might stop to work. Both should be okay.

But perhaps Travis CI will solve this by calling `install-jdk.sh` for us: https://github.com/travis-ci/travis-cookbooks/issues/976 ;)